packages/strapi-plugin-content-type-builder/services/ContentTypes.js

```diff
@@ -9,6 +9,40 @@ const apiHandler = require('./api-handler');
 const { formatAttributes, replaceTemporaryUIDs } = require('../utils/attributes');
 const { nameToSlug, contentTypes: contentTypesUtils } = require('strapi-utils');
 
+const isContentTypeEditable = (contentType = {}) => {
+  const { uid } = contentType;
+  return !uid.startsWith('strapi::') && uid !== 'plugins::upload.file';
```

Comment:
Same as other comments (l.19 & 23 too)

Reply:
Also, why declaring this helper here instead of in `strapi-utils/content-types`?

Reply:
For the moment it's only related to the CTB, I think it would be best to let it here then and maybe move it later if it's used elsewhere in the app?